### ISSUE TITLE
bronk

### ISSUE THREAD

**Issue by fjebaker** [JR-Mitchell/wasmoon-teal-game-template#3]
Following your setup:
```
wasmoon-teal-game-template (main) $ npm run build

> build
> webpack --mode=production

assets by status 349 KiB [cached] 6 assets
runtime modules 7.91 KiB 10 modules
built modules 136 KiB (javascript) 249 KiB (asset) [built]
  modules by path ./ 136 KiB (javascript) 249 KiB (asset)
    modules by path ./src/ 4 KiB (javascript) 1.19 KiB (asset)
      javascript modules 3.92 KiB
        ./src/index.js + 1 modules 3.74 KiB [built] [code generated]
        ./src/assets/ sync ^\.\/.*$ 189 bytes [built] [code generated]
      asset modules 1.19 KiB (asset) 84 bytes (javascript)
        ./src/assets/guy.png 829 bytes (asset) 42 bytes (javascript) [optional] [built] [code generated]
        ./src/assets/icon.png 389 bytes (asset) 42 bytes (javascript) [optional] [built] [code generated]
    modules by path ./node_modules/wasmoon/dist/ 248 KiB (asset) 132 KiB (javascript)
      ./node_modules/wasmoon/dist/glue.wasm 248 KiB (asset) 42 bytes (javascript) [built] [code generated]
      ./node_modules/wasmoon/dist/index.js 132 KiB [built] [code generated]
  url (ignored) 15 bytes [built] [code generated]
  module (ignored) 15 bytes [built] [code generated]

ERROR in unable to locate '/home/lilith/developer/wasmoon-teal-game-template/src/assets/server' glob

webpack 5.96.1 compiled with 1 error in 1277 ms
```
This fixes it enough so I can build, but I doubt it's what you intended:
```diff
diff --git a/webpack.config.js b/webpack.config.js
index 2e932cc..c0bd0c9 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -35,11 +35,6 @@ const config = {
         new HtmlWebpackPlugin({
             template: 'src/index.html'
         }),
-        new CopyWebpackPlugin({
-            patterns: [
-                { from: 'src/assets/server', to: '../server' },
-            ],
-        }),
     ],
     experiments: {
         asyncWebAssembly: true
```

**Comment by fjebaker** [comment]
Also, `server` directory is empty. Is the ignore all in `server/.gitignore` intentional? Means `lua server/server.lua` fails.

**Comment by JR-Mitchell** [comment]
Good catch. This is likely fixed in #2 , in which I realised that I don't need to copy compiled server `.lua` files to a new directory, when you can just directly `tl run` the server `.tl` files.

I'll keep this open until that PR is approved and merged, and then if you can follow the new setup and confirm that it works then I'll close this issue

**Comment by JR-Mitchell** [comment]
I'm assuming this is closed by #2
